### DISCUSSION TITLE
Unable to run setup.py

### DISCUSSION THREAD

**Discussion by cmrodican** [Weather-OS/Fildem-v2#21] [Q&A]
When I get to the part where I must run

`sudo python3 setup.py install --optimize=1
`
the output is as follows:

```
Traceback (most recent call last):
  File "/home/chris/Fildem-v2/setup.py", line 1, in <module>
    import setuptools
ModuleNotFoundError: No module named 'setuptools'

```

Any help is much appreciated.  Thanks in advance!

**Comment by vishnucprasad** [comment]
To install setuptools in debian Run this command

`sudo apt-get install python3-setuptools`
